apax/md/ase_calc.py

```diff
@@ -216,6 +230,40 @@ def calculate(self, atoms, properties=["energy"], system_changes=all_changes):
         self.results = {k: np.array(v, dtype=np.float64) for k, v in results.items()}
         self.results["energy"] = self.results["energy"].item()
 
+    def batch_eval(self, data, batch_size=64, silent=False):
```

Comment:
Perhaps have `data` actually as `atoms_list` since that's what appears to be its role here, unless there are other conventions elsewhere in apax that use `data`? (in which case changes there may also be nice).

Reply:
Probably a minimal docstring since this is a (likely common-use-case) user-facing method.

Reply:
fair points, I"ll adjust the name and add some type hints + docs.